benchcab/data/config-schema.yml

```diff
@@ -44,6 +44,7 @@ realisations:
               revision:
                 type: "integer"
       name:
+        nullable: true
```

Comment:
Why did you add this?

Reply:
Because we validate the yaml after adding in the optional parameters, and `.get()` could return `None`

https://github.com/CABLE-LSM/benchcab/blob/0af9e4e6751a4c0adbaeccba88994697b6b9f8f4/benchcab/config.py#L71